build.gradle

```
@@ -12,9 +13,18 @@ repositories {
 }
 
 group = "io.trocco"
-version = "0.1.0"
 description = "Bigquery output for embulk"
 
+version = {
+    def baseVersion = "0.1.0"
+    def vd = versionDetails()
+    if (vd.commitDistance == 0 && vd.lastTag ==~ /^v[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
+        baseVersion
```

do not use `baseVersion`
```suggestion
        vd.lastTag.substring(1)
```

---

It is a correspondence to link with the release tag.
I got it.

---

fixed https://github.com/trocco-io/embulk-output-bigquery_java/pull/94/commits/1c1d6a09dd51f7a2d0dc9b3fd343daa254782f31